pkg/cmd/scan_ciskubebench.go

```diff
@@ -74,16 +74,19 @@ func ScanKubeBenchReports(cf *genericclioptions.ConfigFlags) func(cmd *cobra.Com
 			go func(node corev1.Node) {
 				defer wg.Done()
 
-				report, err := scanner.Scan(ctx, node)
+				nodeValueLabel, exist := node.GetObjectMeta().GetLabels()["kubernetes.io/os"]
+				if exist && nodeValueLabel == "linux" {
+					report, err := scanner.Scan(ctx, node)
```

Comment:
I think it's even better to avoid creating a gorutine when we detect that's it non linux node. Also we could log warning when we skip Windows nodes:

```go
for _, node := range nodeList.Items {

	if !isLinuxNode(node) {
		klog.Warning("Skipping non linux node: %v %v", node.Name, node.Labels)
		continue
	}

	wg.Add(1)

	// ...
}
```

I prefer exiting the loop asap (using `continue` keyword) to embedding too many `if` statements and shifting the code to the right.